packages/forma-36-react-components/src/components/Card/EntryCard/EntryCard.tsx

```diff
@@ -234,11 +237,11 @@ export class EntryCard extends Component<EntryCardPropTypes> {
                   <div className={styles.EntryCard__body}>
                     {title && this.renderTitle(title)}
                     {description &&
-                      size === 'default' &&
+                      actualSize === 'default' &&
                       this.renderDescription(description)}
                   </div>
                   {thumbnailElement &&
-                    size === 'default' &&
+                    actualSize === 'default' &&
```

Comment:
As part of the story we should be able to show the thumbnail even if the card is small. People rely on images when they search their content.